docs/Data_Portal/Users_Guide/Repository.md

```diff
@@ -42,20 +43,42 @@ If a different filter needs to be used, a custom filter can be applied by choosi
 
 [![Custom File Filter](images/CustomFileFilter.png)](images/CustomFileFilter.png "Click to see the full image.")
 
+## Viewing Images ##
+
+To view images associated with the active cohort, select the View Images button above the files table to launch the Slide Image Viewer. 
+
+## Files Table
+
+The table shows the list of all the files associated with the active cohort, subject to any filtering that has been applied in the Repository. By default, the table provides the following information for each file:
+
+* **Access**: Displays whether the file is open or controlled access. Users must login to the GDC Portal and have the appropriate credentials to access these files.
+* **File Name**: Name of the file. Clicking the link will bring the user to the File Summary Page.
+* **Cases**: The number of cases associated with the file.
+* **Project**: The Project that the file belongs to. Clicking the link will bring the user to the Project Summary Page.
+* **Data Category**: Type of data.
+* **Data Format**: The file format.
+* **File Size**: The size of the file.
+* **Annotations**: Whether there are any annotations.
```

Comment:
This is nit-picky, but having recently reviewed NCI's comments on some of our Encyclopedia pages, they prefer single-sentence bullet-points to _not_ have period at the end (e.g. bullet-points for Cases, Data Category, Data Format, File Size, and Annotations would not have periods)

Reply:
I will fix! Thanks!